Simplenote/Resources/AppStoreStrings.pot

```diff
@@ -63,11 +63,12 @@ msgstr ""
 #. translators: Keywords used in the App Store search engine to find the app.
 #. Delimit with an English comma between each keyword. Limit to 100 characters including spaces and commas.
 msgctxt "app_store_keywords"
-msgid "notes,cloud,simple,notebook,text,to-do,todo,planner,journal,diary,sync,notepad,tag,writing,memo"
+msgid "notes,cloud,simple,notebook,text,to-do,todo,planner,journal,diary,sync,notepad,tag,writing,memo
+"
```

Comment:
This suggestion will need to be applied before merging, or the GlotPress update will fail

```suggestion
msgid "notes,cloud,simple,notebook,text,to-do,todo,planner,journal,diary,sync,notepad,tag,writing,memo"
```

Reply:
@mokagio I think you mentioned there was a scripting issue or something about this. Can you remind me what it is?

Are we going to need to make this change every code freeze? 🤔 

Also, I need to update keywords again, so I'll wait for your answer and do that before we merge this in.

Reply:
I did, but I haven't had the time to track the issue down yet 😞 

Maybe @jkmassel knows? I think he bumped into it, too in the near past.

Reply:
@mokagio I've updated the `keywords` again in a2321612d764c0a167b6f2b9a5a501d6696d3f3b, ran `update_appstore_strings` in d0ffca0dc2b5b48019dae1b37815a72aae5f7bda and removed the extra newline in bae72340ef0dd64c2696d873b4dc28b2f0e6ab37. This is ready for another look!